dcp_prototype/backend/artifact_manager/schemas/v0.0.0.json

```diff
@@ -0,0 +1,408 @@
+                     "required":[
+                        "name",
+                        "institution"
+                     ],
+                     "properties":{
+                        "name":{
+                           "$id":"#/properties/projects/items/properties/contributors/items/properties/name",
+                           "type":"string",
+                           "examples":[
+                              "first last"
+                           ]
+                        },
+                        "institution":{
+                           "$id":"#/properties/projects/items/properties/contributors/items/properties/institution",
+                           "type":"string",
+                           "examples":[
+                              "Stanford"
+                           ]
+                        }
+                     }
+                  }
+               },
+               "protocols_io_doi":{
+                  "$id":"#/properties/projects/items/properties/protocols_io_doi",
+                  "type":"array",
+               },
+               "publication_title":{
+                  "$id":"#/properties/projects/items/properties/publication_title",
+                  "type":"string",
+                  "examples":[
+                     "publication title"
+                  ]
+               },
+               "publication_doi":{
+                  "$id":"#/properties/projects/items/properties/publication_doi",
+                  "type":"string",
+                  "examples":[
+                     "publication doi"
+                  ]
+               },
+               "publication_pmid":{
+                  "$id":"#/properties/projects/items/properties/publication_pmid",
+                  "type":"string",
+                  "examples":[
+                     "publication pmid"
+                  ]
+               },
+               "insdc_project_accessions":{
+                  "$id":"#/properties/projects/items/properties/insdc_project_accessions",
+                  "type":"array",
+               },
+               "geo_series_accessions":{
+                  "$id":"#/properties/projects/items/properties/geo_series_accessions",
+                  "type":"array",
+               },
+               "array_express_accessions":{
+                  "$id":"#/properties/projects/items/properties/array_express_accessions",
+                  "type":"array",
+               },
+               "biostudies_accessions":{
+                  "$id":"#/properties/projects/items/properties/biostudies_accessions",
+                  "type":"array",
+               },
+               "cell_count":{
+                  "$id":"#/properties/projects/items/properties/cell_count",
+                  "type":"integer",
+                  "default":0,
+                  "examples":[
+                     2500
+                  ]
+               }
+            }
+         }
+      },
+      "files":{
```

Comment:
Question about files.   files are currently separated out from projects.   It makes more sense to me to have files be a required item inside the project.  In this way you determine which files are associated with which project.

Reply:
Or we at least need project to be a property of a file, right?

Reply:
I added `project_id` to the file objects. I'm not sure that one project will necessarily _contain_ every file, so I think we can leave it separate for right now.